### PR TITLE
COMMERCE-12590 - Automation for COMMERCE-9599

### PR DESCRIPTION
**Pull request by jaydawu**
Test plan: https://liferay.atlassian.net/browse/COMMERCE-12590

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartProductOptions.testcase [hunk 1]
```diff
@@ -0,0 +1,1961 @@
...
+			Button.clickSave();
+		}
+
+		// The assertion with alert needs to be updated after the COMMERCE-12612 is fixed
+
+	}
+
+	@description = "COMMERCE-12611. As a buyer, I can verify product options work with the decimal quantity correctly"
+	@priority = 5
+	test CanProductOptionsWorkWithDecimalQuantity {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartProductOptions.testcase:1592]: CanBundledProductSkusWorkWithDecimalQuantity

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartProductOptions.testcase [hunk 2]
```diff
@@ -0,0 +1,1961 @@
...
+
+		task ("Then the item for Blue has changed to White with price in the mini-cart") {
+			FrontStore.viewMiniCartItem(
+				bundledItems = "2 × Brake Rotors",
+				cartItemName = "Bundled Product",
+				cartItemOption = "white",
+				cartItemPriceValue = "50.00",
+				cartItemQuantity = 2,
+				cartItemSku = "WHITE");
+
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartProductOptions.testcase:1924]: Here assert that the previous BLUE entry is not present anymore

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
@jaydawu please check the source-formatter failure and the test that is failing in the suite.
Also, if you can change the .testcase file name to `CommerceMiniCartEditBundledProducts.testcase` that would be great!
Thanks!

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:forward

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**